File: official/docs/java/v7/shipments/label.java
package shipments;

import com.easypost.exception.EasyPostException;
import com.easypost.model.Shipment;
import com.easypost.service.EasyPostClient;

import java.util.HashMap;

public class Label {
    public static void main(String[] args) throws EasyPostException {
        EasyPostClient client = new EasyPostClient("EASYPOST_API_KEY");

        HashMap<String, Object> params = new HashMap<String, Object>();
        params.put("file_format", "ZPL");

        Shipment shipment = client.shipment.label("shp_...", params);

        System.out.println(shipment);
    }
}

File: official/docs/node/v7/shipments/label.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient('EASYPOST_API_KEY');

(async () => {
  const shipment = await client.Shipment.convertLabelFormat('shp_...', 'ZPL');

  console.log(shipment);
})();
